pages/index.js

```diff
@@ -90,6 +105,26 @@ export default function Home({ locale, popularCategories, situationCookie }) {
     setBenefits(data);
   }, [situation]);
 
+  let locationAssumed = false;
+  //only make location assumptions if the situation cookie doesn't have anything set for location
+  if (situation?.location === undefined) {
+    locationAssumed = true;
+    assumeLocation()
+      .then((location) => {
+        setSituation((previousState) => ({
+          ...previousState,
+          ["location"]: location,
+        }));
+        document.getElementById("location-select").value = location;
+        document.getElementById("location-assumption").innerHTML =
```

Comment:
Where this is displayed we should have a prop passed in that either shows or hides this message.
Check out PopularCategoryCard.js and how props.imgSource changes how the component displays